corehq/apps/translations/migrations/0001_initial.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-12-11 09:52
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TransifexOrganization',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('slug', models.CharField(max_length=255)),
+                ('name', models.CharField(max_length=255)),
+                ('api_token', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TransifexProject',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('slug', models.CharField(max_length=255)),
```

Comment:
should this be unique?

Reply:
Good point. I don't see people sharing projects but ya good to add to avoid them.